### ISSUE TITLE
SPIKE: Determine the best approach for incorporating thumbnails into Blacklight

### ISSUE THREAD

**Issue by lovinscari** [emory-libraries/blacklight-catalog#69]
As the product owner, I would like to explore different approaches in an effort to determine the best option for incorporating thumbnails into Blacklight that will be used when displaying search results and single item details. Currently, Primo uses Syndetic Solutions (Proquest company,  which is free for Primo customers. However, as Emory plans to move away from Primo over the next few years, this will not be a sustainable solution. Google Books has been mentioned as an option (free, not as complete or reliable), and possibly Amazon. 

- [x] The ideal solution for thumbnails will provide a large range of thumbnail images displayed for a minimum of 70% of all resources currently in the catalog.

- [x] The ideal solution will be free or low cost that is easily integrated into Blacklight.

- [x] The ideal solution will provide thumbnails in color that provide enough resolution to be easily recognized as the cover for a resource without degrading page load times. 

- [x] An ideal solution will provide generic thumbnails for different resource types where no cover artwork is available.

**Comment by rotated8** [comment]
- Does the solution have to be free? (Could we buy a Syndetic subscription?)
- Time boxing this to a day.

**Comment by bwatson78** [comment]
Attached is a report accumulating all of the options I've found. Our Business library currently utilizes Google Books for thumbnails, and I recommend the continuation of that practice. I have read in articles posted by university library software engineers that they prefer to query both OpenLibrary and Google Books APIs (in that order) to increase their chances of getting reliable images. We could do the same. Syndetics doesn't appear to be an option for us since they provide no API interface, reliant on the Primo ecosystem, and costs $696 a year for just cover images.

As far as Amazon is concerned, I came across articles that universities have run into legal trouble for utilizing their images and have been denied API access based upon their intended use.

[https://app.zenhub.com/files/295449627/9219f130-81aa-4d44-b59b-04074918bff5/download]
